eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/CCSubClient.java

```diff
@@ -44,7 +44,7 @@ public static void main(String[] args) throws Exception {
         subClient.registerBusiHandler(new ReceiveMsgHook() {
             @Override
             public void handle(Package msg, ChannelHandlerContext ctx) {
-                logger.error("Received message: -----------------------------------------" + msg.toString());
+                logger.error("Received message: {}", msg);
```

Comment:
suggest implement Package class  toString() method.

Reply:
thanks，i will try it

Reply:
Package class has @Data annotation which has implemented toString() method.